app-backend/db/src/test/scala/com/azavea/rf/database/CategoryToTemplateDaoSpec.scala

```diff
@@ -11,5 +11,7 @@ import doobie.scalatest.imports._
 import org.scalatest._
 
 class CategoryToTemplateDaoSpec extends FunSuite with Matchers with IOChecker with DBTestConfig {
-  // test("selection types") { check(CategoryToTemplateDao.selectF.query[CategoryToTemplate]) }
+  test("selection types") {
+    CategoryDao.selectF.query[Category]
```

Comment:
This isn't quite right (and is the thing I'm crankiest about with doobie -- `aFragment.query` transforms from a fragment to a `Query0` or something, which then gets transformed into a `ConnectionIO` if you throw a `.unique` or a `.option` or whatever on the end... but no typechecking occurs against what's in the db until you go to the db and try to execute it. That's why I suggested `...list.transact(xa).unsafeRunSync` before -- I've tested to make sure that the result of that query has a length >= 0 in other places, because it's a way to go to the db once and make sure that the query is successful without having to have doobie's `check` and scalatest's `check` in the same zone

Reply:
By "not quite right" I mean -- I don't think this is testing anything. I think you could put in an arbitrary datatype in the `.query[Foo]` and it would pass

Reply:
Per discussions offline, we plan to take care of it when doing property tests for category and template dao instead of in this card for workspace dao.